addons/mrp/models/mrp_workorder.py

```diff
@@ -233,6 +259,12 @@ def _compute_allowed_lots_domain(self):
                 treated |= workorder
         (self - treated).allowed_lots_domain = False
 
+    def action_replan(self):
+        productions = self.mapped('production_id')
+        for production in productions:
+            production._plan_workorders(replan=True)
```

Comment:
for production in self.'production_id:
            production._plan_workorders(replan=True)